### PR TITLE
Removed a wrong note about controllers as services

### PR DESCRIPTION
**Pull request by javiereguiluz**
This is probably wrong. We're discussing about that in symfony/symfony#24976.

### CODE REVIEW

**Comment by xabbuh** [comment]
I would reword the note a bit. Its initial goal was to say that you cannot drop the `Action` suffix in the configuration when it is part of your method name (which is different from what we have when using the triple syntax). I think we should write something like that you must give the full method name and that no `Action` will be suffixed automatically.

**Comment by chalasr** [comment]
Agree with @xabbuh

**Review by yceruto — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

File: controller/service.rst [hunk 1]
```diff
@@ -102,6 +102,12 @@ defined above with the id ``app.hello_controller``::
 
     $this->forward('app.hello_controller:indexAction', array('name' => $name));
 
+.. note::
+
+    Unlike the traditional ``Bundle:Controller:method`` notation, when using
+    this syntax the method name must match the original controller method name,
+    so you cannot remove the ``Action`` suffix if the method includes it.
```
Comment (xabbuh) [inline controller/service.rst]: I would make this a new sentence

**Review by xabbuh — APPROVED** [review]

File: controller/service.rst [hunk 2]
```diff
@@ -104,8 +104,9 @@ defined above with the id ``app.hello_controller``::
```
Comment (weaverryan) [inline controller/service.rst]: Unrelated, but I think we should remove the `forward` example entirely and just keep the routing example (and probably move the note below the routing example).

File: controller/service.rst [hunk 3]
```diff
@@ -104,8 +104,9 @@ defined above with the id ``app.hello_controller``::
 
 .. note::
 
-    You cannot drop the ``Action`` part of the method name when using this
-    syntax.
+    Unlike the traditional ``Bundle:Controller:method`` notation, when using
+    this syntax the method name must match the original controller method name,
+    so you cannot remove the ``Action`` suffix if the method includes it.
```
Comment (weaverryan) [inline controller/service.rst]: Maybe this wording?

```rst
Make sure the method name in your route (e.g. ``indexAction``) matches the method name exactly.
Unlike the traditional ``Bundle:Controller:method`` notation, the `Action` suffix is not automatically
added for you.
```

**Review by weaverryan — CHANGES_REQUESTED** [review]

File: controller/service.rst (hunk 3 above)
Comment (javiereguiluz) [reply controller/service.rst]: I like your proposal. Much better. Thanks!

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.
